### DISCUSSION TITLE
Customer Addresses are not being updated

### DISCUSSION THREAD

**Discussion by jorge-braz** [Shopify/storefront-api-feedback#221] [Help]
I have just encountered a problem with Storefront API. 
Whenever I delete a a customer address, the address list is not being updated. So all the deleted addresses still show up.

The address list is updated on shopify back-office and if I make a request to the shopify admin api, unfortunately not on the storefront.

Anyone else with this problem?

**Comment by johncraigcole** [comment]
Hey @jorge-braz we have an issue internally for this. Don't have a timeline yet on resolution, but will update here when it ships.

**Reply by johncraigcole** [reply]
Hey @jorge-braz this issue should now be resolved.

**Reply by jorge-braz** [reply]
I confirm. Thank you
